lib/iris/fileformats/pp.py

```diff
@@ -1040,7 +1040,8 @@ def _data_bytes_to_shaped_array(data_bytes, lbpack, boundary_packing,
 
     # Mask the array?
     if mdi in data:
-        data = array_masked_to_nans(data, data == mdi)
+        data = np.ma.masked_array(data, data == mdi)
```

Comment:
This allows the `mask` kwarg to be dropped from `array_masked_to_nans`. (This is also the only place the kwarg was used!)